flocker/apiclient/_client.py

```diff
@@ -27,8 +27,10 @@
 
 from treq import json_content, content
 
+# XXX These dependencies all make splitting the client out into a separate
+# package harder.
```

Comment:
:grey_question: I'd prefer not to add this `XXX`. I'm not sure if it adds much value, and `TODOs` make me sad (they can get stale, they can be a bit overwhelming if your code is full of them, etc.).

Is it supposed to deter people from adding dependencies? That seems like the wrong answer--we don't want unnecessary duplication. Things that are added here presumably should be put in a separate package that both flocker and the client library would have as a dependency, regardless of how many there are.

Optional as I recognize this is not the most objective view of the world, but I'd prefer not adding this particular `XXX`.